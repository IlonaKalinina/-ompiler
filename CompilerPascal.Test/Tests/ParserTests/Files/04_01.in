program Test_04_01;
var
a : record
x : string;
y : integer;
end;
begin
a.y := 10;
end.