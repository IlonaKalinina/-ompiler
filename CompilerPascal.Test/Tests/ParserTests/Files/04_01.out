program Test_04_01
├───var
│    └─── :
│         ├─── a
│         └─── record 
│              ├─── x
│              │    └─── string
│              └─── y
│                   └─── integer
│    
begin
├─── :=
│    ├─── .
│    │     ├─── a
│    │     └─── y
│    └─── 10
└─── end
